### PR TITLE
chore: fixed deprecation errors for dropdown labels

### PR DESCRIPTION
**Pull request by jpvajda**
This PR: 

1. fixes general prettier Linter errors.
2. fixes eslint errors
3.  removes label from dropdown components as those are now deprecated
4. Updates package-lock.json
5. improved the UX for the Setup dropdown selectors...

Screen Shot 2021-02-12 at 4 13 38 PM

### CODE REVIEW

File: nerdlets/nr1-cloud-migrator-nerdlet/components/complexity-profiler/index.js [hunk 1]
```diff
@@ -232,7 +232,7 @@ export default class ComplexityProfile extends React.PureComponent {
               summary.languages = [...new Set(languages)];
               summary.guids = [...new Set(guids)];
 
-              console.debug("summary obj", summary);
+              console.debug('summary obj', summary);
```
Comment (caylahamann) [inline nerdlets/nr1-cloud-migrator-nerdlet/components/complexity-profiler/index.js:235]: out of curiosity, why is this here?

**Review by caylahamann — APPROVED** [review]
loooking good! just curious about the `console.debug` statement

File: nerdlets/nr1-cloud-migrator-nerdlet/components/complexity-profiler/index.js (hunk 1 above)
Comment (jpvajda) [reply nerdlets/nr1-cloud-migrator-nerdlet/components/complexity-profiler/index.js:235]: @caylahamann probably not for the user... I can remove it.  Maybe it was just a troubleshooting step someone forgot to remove?

Comment (jpvajda) [reply nerdlets/nr1-cloud-migrator-nerdlet/components/complexity-profiler/index.js:235]: Actually this file has a series of console.log statement as well. so not knowing why, I'm thinking it's better to leave it in. Let me know what you think.
